_episodes/02-vcs.md

```diff
@@ -18,7 +18,7 @@ keypoints:
 - "VCS can be used directly or can serve a foundation for domain-specific tools"
 ---
 
-> ## You can skip this lesson if you can answer these questions --->
+> ## You can skip this lesson if you can answer these questions
```

Comment:
yeah, probably no need for those arrows since web UI shows the "drop down" arrow.  But let's add `:` for consistency:
```suggestion
> ## You can skip this lesson if you can answer these questions:
```